packages/project-access/src/project/cap.ts

```diff
@@ -96,3 +95,15 @@ export async function getCapModelAndServices(
         services
     };
 }
+
+/**
+ * Get CAP CDS project environment config for project root.
+ *
+ * @param capProjectPath - project root of a CAP project
+ * @returns - environment config for CAP project
+ */
+export async function getCapEnvironment(capProjectPath: string): Promise<CdsEnvironment> {
+    const cds = await loadModuleFromProject<any>(capProjectPath, '@sap/cds');
```

Comment:
`unknown` would be the better type here.

Reply:
If I use `unknown` then we get compiler error in the next line. We would need a specific type instead, I suppose we could do that as well.

Reply:
Do you know the type, then yes, please. Based on the next line, I assumed you don't know exactly what you get.